docs/security.rst

```diff
@@ -280,3 +280,65 @@ If you have your own HBA provisioning solution, you can include the rules
 needed for pg_auto_failover and then use the ``--skip-pg-hba`` option to the
 ``pg_autoctl create`` commands.
 
+
+Enable SSL connections on an existing setup
+-------------------------------------------
+
+Whether you upgrade pg_auto_failover from a previous version that did not
+have support for the SSL features, or when you started with ``--no-ssl`` and
+later change your mind, it is possible with pg_auto_failover to add SSL
+settings on system that has already been setup without explicit SSL support.
+
+In this section we detail how to upgrade to SSL settings.
+
+Installing Self-Signed certificates on-top of an already existing
+pg_auto_failover setup is done with one of the following pg_autoctl command
+variants, depending if you want self-signed certificates or fully verified
+ssl certificates::
+
+  $ pg_autoctl enable ssl --ssl-self-signed --ssl-mode required
+
+  $ pg_autoctl enable ssl --ssl-ca-file root.crt   \
+                          --ssl-crl-file root.crl  \
+                          --server-crt server.crt  \
```

Comment:
Throughout the documentation we use `--server-crt` but the option is `--server-cert`. This should be fixed in many more places.
```suggestion
                          --server-cert server.crt  \
```